packages/genie/package.py

```diff
@@ -92,9 +111,9 @@ def configure_args(self):
         if self.spec.satisfies("@3.00.00:"):
             args.extend(
                 [
-                    "--enable-lhapdf5",
-                    "--with-lhapdf5-lib={0}".format(self.spec["lhapdf"].prefix.lib),
-                    "--with-lhapdf5-inc={0}".format(self.spec["lhapdf"].prefix.include),
+                    "--enable-lhapdf6",
+                    "--with-lhapdf6-lib={0}".format(self.spec["lhapdf"].prefix.lib),
+                    "--with-lhapdf6-inc={0}".format(self.spec["lhapdf"].prefix.include),
```

Comment:
Is this true for the whole version range of the conditional?

Reply:
Working on this a little.  Even making  lhapdf an optional (default true) dependency., and using the lhadf6 versions when its an lhapdf:@6: dependency.